packages/server/modules/core/graph/resolvers/projects.ts

```diff
@@ -158,7 +180,15 @@ export = {
       for (const batch of inputBatches) {
         await Promise.all(
           batch.map((i) =>
-            createStreamInviteAndNotify(
+            createStreamInviteAndNotify({
```

Comment:
createStreamInviteAndNotify is a factory too and thus should have the prefix. Every HOC that injects deps should have the Factory prefix, not just deps of top-level factories